docs/integrations/sources/adroll.md

```diff
@@ -0,0 +1,52 @@
+# AdRoll
+Docs : https://developers.adroll.com/apis
```

Comment:
Ideally, we should have a one-sentence summary of what the hell adroll is, and what kind of data are we syncing back, what APIs we're using. For all links, we should have actual markdown links, not just the urls.